docker-compose.yml

```
@@ -0,0 +1,65 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      args:
+        user: opendor-dev
+        uid: 1000
+      context: ./
+      dockerfile: .docker/php/Dockerfile
+    image: php:8.0
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - ./:/var/www
+    depends_on:
+      - db
```

Both are needed to run the app.
```suggestion
    depends_on:
      - db
      - redis
```